docs/README.md

```diff
@@ -0,0 +1,219 @@
+For simplicity, create a GitHub repository and add a single main package:
+```go
+// main.go
+package main
+
+func main() {
+  println("Ba dum, tss!")
+}
+```
+By default, GoReleaser will build the *main.go* file located in your current directory, but you can change the build package path in the GoReleaser configuration file.
+
+```yml
+# goreleaser.yml
+# Build customization
+build:
+  binary_name: drum-roll
+  goos:
+    - windows
+    - darwin
+    - linux
+  goarch:
+    - amd64
+```
+
+This configuration specifies the build operating systems to Windows, Linux and MacOS using 64bit architecture, the name of the binaries is "drum-roll".  
+
+GoReleaser will then archive the result binaries of each Os/Arch into a separate file. The default format is `{{.BinaryName}}_{{.Os}}_{{.Arch}}`.  
+You can change the archives name and format. You can also replace the OS and the Architecture with your own.  
+Another useful feature is to add files to archives, this is very useful for integrating assets like resource files.
+
+```yml
+# goreleaser.yml
+# Build customization
+build:
+  main: main.go
+  binary_name: drum-roll
+  goos:
+    - windows
+    - darwin
+    - linux
+  goarch:
+    - amd64
+# Archive customization
+archive:
+  format: tar.gz
+  replacements:
+    amd64: 64-bit
+    darwin: macOS
+    linux: Tux
+  files:
+    - /path/drum-roll.licence.txt
+```
+
+This configuration will generate tar archives, contains an additional file "drum-roll.licence.txt", the archives will be  located in:  
+  "./dist/drum-roll_windows_64-bit.tar.gz"  
+  "./dist/drum-roll_macOS_64-bit.tar.gz"  
+  "./dist/drum-roll_Tux_64-bit.tar.gz"  
+
+Next, you need to export a `GITHUB_TOKEN` environment variable with the `repo` scope selected. This will be used to deploy releases to your GitHub repository. Create yours [here](https://github.com/settings/tokens/new).  
+```sh
+export GITHUB_TOKEN=`YOUR_TOKEN`
+```
+
+GoReleaser uses the latest [Git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) of your repository.
+Create a tag:  
+```sh
+git tag -a v0.1 -m "First release"
+```
+
+Now you can run GoReleaser at the root of your repository:  
+```sh
+curl -s https://raw.githubusercontent.com/goreleaser/get/master/latest | bash
+```
+
+That's it!, check your GitHub release page.  
+The release on GitHub looks pretty much like this:
+
+[![image](https://cloud.githubusercontent.com/assets/245435/21578845/09404c8a-cf78-11e6-92d7-165ddc03ca6c.png)
+](https://github.com/goreleaser/goreleaser/releases)
+
+## Environment setup
+
+### GitHub Token
+GoReleaser requires a GitHub API token with the `repo` scope checked to deploy the artefacts to GitHub. You can create one [here](https://github.com/settings/tokens/new).  
+This token should be added to the environment variables as `GITHUB_TOKEN`. Here is how to do it with Travis-ci: [Defining Variables in Repository Settings](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings).  
```

Comment:
I think we have only one section for it. Maybe you ment the Quick start part about tokens.
If so, then I don't think it's a duplication since it doesn't explain the token properly.